### PR TITLE
 Master to main

### PR DESCRIPTION
**Pull request by ngranda**
Change master to main in templates. Do the docker images need to be updated for this change to be right?

### CODE REVIEW

File: cpp_config_template.yml [hunk 1]
```diff
@@ -21,7 +21,7 @@ jobs:
                 docker:
                         - image: cnerg/ubuntu-18.04_pyne-3:latest #any public docker container
                 steps:
-                        -run: git push origin master
+                        -run: git push origin main
```
Comment (gonuke) [inline cpp_config_template.yml]: I agree with this change, but I'm not sure I think it's wise to have a `pushing` job in this config file.  It seems either dangerous or unnecessary, or both - especially for the beginners for whom these templates are intended.

Perhaps @psbritt can remind us why it's here?

**Review by gonuke — CHANGES_REQUESTED** [review]
Requesting a change that is not really related to this PR... sorry.  Pinging @psbritt

**Comment by psbritt** [comment]
It’s been a couple of months since we went over this, but isn’t the point of CI that the multiple versions of the code being worked on by different people be periodically tested and then merged into a single common location - i.e. being pushed to the git repository that everyone shares?

**Comment by gonuke** [comment]
The push is typically a manual step.  CI usually just ensures that tests pass, but manual code review is necessary before merging/pushing.

**Comment by psbritt** [comment]
﻿
In that case, I’ll remove it.

**Comment by gonuke** [comment]
@ngranda - please rebase this with PR #11

**Comment by ngranda** [comment]
> @ngranda - please rebase this with PR #11

done!

**Comment by gonuke** [comment]
Thanks @ngranda

**Review by gonuke — APPROVED** [review]

**Pull request merged**